VirtualEconomyFramework/VENFTApp-Blazor/Pages/Send.razor

```diff
@@ -514,11 +514,11 @@
 
                         @if (mainAccountSelected)
                         {
-                            <VENFTApp_Blazor.Components.Account.TransactionsList SubAccount="false" Address="@AppData.Account.Address" @bind-MaxLoadedTx="maxLoadedTransaction" />
+                            <VENFTApp_Blazor.Components.Account.TransactionsList SubAccount="false" Address="@AppData.Account.Address" />
                         }
                         else if (!mainAccountSelected && !string.IsNullOrEmpty(selectedAccountAddress))
                         {
-                            <VENFTApp_Blazor.Components.Account.TransactionsList SubAccount="true" Address="@selectedAccountAddress" @bind-MaxLoadedTx="maxLoadedTransaction" />
+                            <VENFTApp_Blazor.Components.Account.TransactionsList SubAccount="true" Address="@selectedAccountAddress" />

```

Comment:
We passed `MaxLoadedTx` in multiple files with the same value. I hardcoded it for now